packages/commonwealth/server/scripts/setupChainEventListeners.ts

```diff
@@ -174,10 +174,12 @@ const setupChainEventListeners = async (
           });
         } else if (node.network === ChainNetwork.Moloch) {
           const contractVersion = 1;
+          // @TODO: @JAKE @CONTRACTS: fetch governance contract specifically
```

Comment:
TODO -- not sure what we should do with this file. May be deprecated post-Timothee's changes.

Reply:
I think this is definitely a post-timothee change.

Reply:
@timolegros

Reply:
This file is deleted in the Chain-Events microservice PR.

Reply:
@zakhap is this PR blocked on CE Microservice PR then?

Reply:
i don't use this file in any meaningful way in my PR, was just updating it to "work right" with current chain events. being that communities only have 1 contract by default, this should have no effect. I can be redundantly clear at fetching the contract that the contract needs to be of a particular type. Will do so